src/operator/bufferTime.ts

```diff
@@ -168,13 +168,13 @@ class BufferTimeSubscriber<T> extends Subscriber<T> {
     this.contexts = null;
   }
 
-  protected onBufferFull(context: Context<T>) {
+  protected onBufferFull(context: Context<T>): void {
     this.closeContext(context);
     const closeAction = context.closeAction;
     closeAction.unsubscribe();
     this.remove(closeAction);
 
-    if (this.timespanOnly) {
+    if (this.timespanOnly && this.contexts !== null) {
```

Comment:
this is actual changes, do not attempt to open buffer if it's already unsubscribed.